server/src/controllers/index.js

```diff
@@ -0,0 +1,5 @@
+const user = require('./UsersController');
+
+module.exports = {
+  user
+};
```

Comment:
```suggestion
const userController = require('./UsersController');

module.exports = {
  userController
};
```

Trzymamy się konwencji nazewniczej.

Reply:
done